baudRate: 1000000
port: /dev/ttyUSB0
servoIds: [1, 6]